pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -2,40 +2,47 @@
 , stdenv
 , fetchpatch
 , kernel
-, commitDate ? "2023-06-28"
 # bcachefs-tools stores the expected-revision in:
 #   https://evilpiepirate.org/git/bcachefs-tools.git/tree/.bcachefs_revision
 # but this does not means that it'll be the latest-compatible revision
-, currentCommit ? "84f132d5696138bb038d2dc8f1162d2fab5ac832"
-, diffHash ? "sha256-RaBWBU7rXjJFb1euFAFBHWCBQAG7npaCodjp/vMYpyw="
+, version ? lib.importJSON ./bcachefs.json
```

Comment:
Given we might soon get rid of this kernel, it might be not worth it to have complex update logic here. Some hashes here might be enough.